### PR TITLE
DateInput - Adiciona no input a data do preset pré-selecionado

### PR DESCRIPTION
**Pull request by vagnervst**
## Context
Ao utilizar o DateInput com a prop `selectedPreset`, o preset é selecionado mas a sua data não é inserida no input de texto. Este PR soluciona este problema.

## Checklist
- [x] Ao utilizar a prop `selectedPreset`, a data do range do preset deve ser inserida nos inputs de texto
- [x] Adicionada a prop `isValidDay` no DateInput e no DateSelector para podermos bloquear dias não úteis
- [x] Corrige a validação de props do Moment

**Solves:** https://github.com/pagarme/former-kit/issues/235

### CODE REVIEW

File: src/DateInput/index.js [hunk 1]
```diff
@@ -60,10 +60,18 @@ class DateInput extends React.Component {
   constructor (props) {
     super(props)
 
+    const presetObject = props.selectedPreset
+      ? getPreset(props.selectedPreset, props.presets)
+      : null
+
     this.state = {
-      dates: momentToText(props.dates),
+      dates: presetObject
```
Comment (rsmelo) [inline src/DateInput/index.js]: I think is better for readability if you set `const` before passing it to state as you did in line 63

```js
const dates =  presetObject
        ? momentToText(getPresetLimits(presetObject.date()))
        : momentToText(props.dates)
```
and 
```js
const selectionMode = presetObject
        ? presetObject.mode
        : props.selectionMode
```

Comment (vagnervst) [reply src/DateInput/index.js]: I agree! Changes applied.

File: src/Calendar/index.js [hunk 2]
```diff
@@ -51,6 +52,16 @@ const isOutsideRange = limits => complement(validateDate(limits))
...
   times(add(currentMonth), months)
 
+export const isMomentPropValidation = (props, propName) => {
+  const propValue = props[propName]
+
+  if (propValue && !moment.isMoment(propValue)) {
+    return new Error(`Prop ${propName} must be an instance of Moment`)
+  }
+
+  return null
```
Comment (uselessdev) [inline src/Calendar/index.js]: amaze 🎉

**Comment by rsmelo** [comment]
maybe this commit (e3c447081fe6c198e33d8dae16b6bbcd2f78b896) message can use `calendarInput` context, my suggestions is `calendarInput: fix isValid date format`

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fb8de600c01bc15b8082e7f2caca1d7e7663b726
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fb8de600c01bc15b8082e7f2caca1d7e7663b726
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by vagnervst** [comment]
@rsmelo Done!

**Pull request merged**